scripts/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# This script builds the networkservicemesh
+#
+
+go get -u github.com/golang/protobuf/protoc-gen-go
+go generate ./...
+go install ./...
+go test -race ./...
```

Comment:
So it turns out using "go test -race" with alpine linux is a non-starter [1]. I am going to remove this for now.

[1] https://github.com/golang/go/issues/14481